portfolio/src/main/java/com/google/sps/functionality/TranslateText.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.functionality;
+
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+
+/** Class that translates text.  */
+public class TranslateText { 
```

Comment:
if the class translates text, perhaps a better name could be "TextTranslator". 

In general, class names should be descriptive of objects, and should thus be named like Nouns. Where as TranslateText is more of a verb sentence fragment. 

Preferring Nouns for class names and Verbs for methods enables a sentence-like structure of the underlying code. 

e.g.
if(languageParam.isPresent) {
  textTranslator.translate();
}

Reply:
Class names should be nouns, not verbs, got it! Fixed :)